src/main/java/com/epam/spring/homework2/Application.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.spring.homework2;
+
+import com.epam.spring.homework2.config.Config;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import java.util.Arrays;
+
+public class Application {
+    public static void main(String[] args) {
+        System.out.println("############### Starting context... ###############");
+        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(Config.class);
+        System.out.println("############### List of all beans ###############");
+        Arrays.stream(context.getBeanDefinitionNames()).forEach((s) -> System.out.println(context.getBean(s)));
```

Comment:
I agree, that 12th task looks a little bit confusing ))) Please use '**getBeanDefinition**' method to display Bean Definition instead of '**getBean**'